test-newer-jdks/orchard/java/parser_test.clj

```diff
@@ -1,66 +1,50 @@
 (ns orchard.java.parser-test
   (:require
+   [clojure.test :refer [deftest is testing]]
    [orchard.java.parser :as parser]
-   [clojure.test :refer [deftest is testing]]))
+   [orchard.test.util :as util])
+  (:import
+   (orchard.java DummyClass)))
 
-(defn compile-class-from-source
```

Comment:
Now compilation for this test case happens via Lein :java-source-paths, it reflects typical end-user usage and also exercises what enrich-classpath does / is able to do (it's not able to modify classpaths at runtime, and we don't want that either!)